### PR TITLE
added compass and UI flourishes

### PR DESCRIPTION
**Pull request by carriebarrett**
Added compass to map view, added logo to app bars, reorganized image assets, cleaned up code smells (commented out code, style guide, etc.)

5/6 update: 
-Added constants file
-Changed app bar image paths to use constant instead of string literal
-Moved map view build method to the end of the class (for style)
-Added whitespace to getLocation() for readability
-Changed order of imports to fit dart style guide (see [https://dart.dev/guides/language/effective-dart/style])

### CODE REVIEW

File: lib/screens/map_view.dart
Comment (jorellana28) [inline lib/screens/map_view.dart]: What does this widget do?

File: lib/screens/map_view.dart
Comment (jorellana28) [inline lib/screens/map_view.dart]: If we are going to reuse this logo in multiple places, can we make it a global variable we store somewhere?

File: pubspec.yaml
Comment (jorellana28) [inline pubspec.yaml:37]: Can we just do something like assets/images/*? Or does that not work

**Review by jorellana28 — COMMENTED** [review]
I just added a few comments

File: lib/screens/map_view.dart
Comment (carriebarrett) [reply lib/screens/map_view.dart]: It's what allows the beastie to still be clickable after adding the compass layer above it.

File: lib/screens/map_view.dart
Comment (carriebarrett) [reply lib/screens/map_view.dart]: That's a good idea, it would definitely make it easier to use

File: pubspec.yaml
Comment (carriebarrett) [reply pubspec.yaml:37]: That doesn't work in flutter unfortunately :(

File: lib/screens/map_view.dart
Comment (carriebarrett) [reply lib/screens/map_view.dart]: Do you want me to make that change before approval? EDIT: I went ahead and made the change

**Review by jorellana28 — APPROVED** [review]
Sorry I didn't see the replies! If I ever don't respond feel free to ping me in teams ☺️

**Pull request merged**
